Inventory.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace InventoryManagementSystem
+{
+    public class Inventory
+    {
+        private List<Product> listOfProducts = new List<Product>();
```

Comment:
Second role in Clean code, the name of method or property or class should be have the meaning of what it is, and when the type is List we don't need to add "list" in suffix or prefix, 
What I mean here is: that you don't need to add listOf only naming it as the plural name of product  = "Products"
```suggestion
        private List<Product> Products = new List<Product>();
```